dll/win32/browseui/CAutoComplete.cpp

```diff
@@ -272,17 +272,21 @@ EditWordBreakProcW(LPWSTR lpch, INT index, INT count, INT code)
         {
             if (index)
```

Comment:
Note for @katahiromz (iirc you wrote this code?)
The function should check whether index is >= count and if so, quite probably either fail, or have an assert.